sdk/batch/Microsoft.Azure.Batch.FileStaging/src/Microsoft.Azure.Batch.FileStaging.csproj

```diff
@@ -22,6 +22,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.Text.Encodings.Web" VersionOverride="4.5.1" />
```

Comment:
Should we rev the global track 1 `System.Text.Encodings.Web` version?

Reply:
Is there such a thing? The only reason I used `VersionOverride` here is because enough of the common targets err if you use `Version`, but the only "System.Text.Encodings.Web" I found in _eng/Packages.Data.props_ was for track 2 and other newer packages (4.7.2).

/cc @benbp @weshaggard @chidozieononiwu

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Packages.Data.props#L10-L13

We might need to add an entry.

Reply:
Yes lets try to use the entry in Packages.Data.props.

Reply:
Thanks. I'll try. It may be a little tricky, seeing as only 4.0.0-4.5.0 should update to 4.5.1. Can we make an assumption that that conditioned item group should only use 4.5.1?

Reply:
Is there any reason we cannot pull everyone up to the latest 4.7.2? If we cannot then I think as long as it is correctly scoped under the right item group we should be fine.

Reply:
That brings other newer dependencies that could lead to TypeLoadExceptions and such. Mitigations recommend 4.0-4.5.0 use 4.5.1.